team_minutes/2020-07-24.md

```diff
@@ -234,7 +234,7 @@ R Consortium Diversity & Inclusion Group: Samantha Toet, gwynn sturdevant, Hadle
         - Groups formed
         - September to assess which issues have volunteers and projects
         - Feel free to get in touch with comments later
-    - Many thanks to HT for organizing from ST, AM, JB, EC, AV, RB, SKB, MCR, NT, GR, GQ, YB, CW, MG, ER
+    - Many thanks to HT for organizing from ST, AM, JB, EC, AV, RG, SKB, MCR, NT, GR, GQ, YB, CW, MG, ER
```

Comment:
thanks @emdodwell for the corrections above, any idea on this one?

Reply:
RG is the right correction. My goodness, I was exhausted when I incorporated the chat into the minutes and apparently completely forgot how the alphabet works -- I can do a quick spot check in its entirety tomorrow. Thanks for catching these, @MichaelChirico.

Reply:
i promise it's better than i'd do 😄 thanks again for transcribing